+bisos/+iteration/+step/bisect.m

```diff
@@ -31,7 +31,7 @@
         
         stepsol = goptimize(sosc,objective,sosoptions);
         
-        info.subprob.iter = stepsol.subiter;
+        info.bisections = stepsol.subiter;
```

Comment:
What's the purpose of `stepsol.subiter`? From some runs the value of this output has been null, is it what it is expected or the goal here is to save data from the bisections operations, in other words instead of `stepsol.subiter`, it would be `stepsol.solverinfo` which has for example the number of bisection operations done

Reply:
Yes, it is the number of bisections. The value in `stepsol.solverinfo` is the number of iterations that Sedumi has performed.